README.md

```diff
@@ -99,7 +99,7 @@ The environment variable that is set when using the `track` `internalsharing`
 ## Example usage
 
 ```yaml
-uses: r0adkll/upload-google-play@v1
+uses: swootte/upload-google-play@V1.1.5
```

Comment:
The example usage of this repo won't have an example using another repo... and there's no reason to have it reference a specific subversion either